tox.ini

```diff
@@ -24,3 +22,6 @@ deps =
     pep8==1.6.2
 commands =
     flake8 cookiecutter
+
+[testenv:cov-report]
+commands = py.test --cov=cookiecutter --cov-report=term --cov-report=html
```

Comment:
Seeing the way you're using `--cov` here, makes me think that https://github.com/hackebrot/cookiecutter/blob/remove-test-command-in-favor-of-tox/tox.ini#L6 is incorrect?